app-rails/config/environments/production.rb

```diff
@@ -75,9 +75,13 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Use a real queuing backend for Active Job (and separate queues per environment).
-  # config.active_job.queue_adapter = :resque
-  # config.active_job.queue_name_prefix = "app_rails_production"
```

Comment:
Do you think we need a `queue_name_prefix`?

Reply:
You have a lot more Rails expertise than I do so please LMK if this is the wrong line of thinking, but since jobs are queued using a table in the DB I don't think that we'd need to worry about a prefix since we won't be sharing the database across environments? I would think that the queue prefix might be more necessary if we were doing something like sharing a Redis instance across multiple environments, for example.

Reply:
After looking at this and thinking about it more, I think we should move this line:

```ruby
  config.active_job.queue_adapter = :good_job
```

out of the per-environment files and into `/app-rails/config/application.rb`. What do you think? Good Jobs should definitely be used in the `mock-production` environment. I'm not sure what it might do to the test environment, though. Thoughts?